src/amo/components/Category/styles.scss

```diff
@@ -0,0 +1,3 @@
+.Category .ErrorList {
```

Comment:
Because `.ErrorList` is shared and overridden in so many places we should be aware that it's going to eventually have some possibly weird cascades. This is fine for now and nothing that needs refactoring, just something to remain vigilant about 😄